suite-native/module-settings/src/components/CurrencySelector.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Can you try add tests for this component? You don't need to check what is in redux state, but you should check that if select CZK for example, that there is text CZK in Select as label.